### ISSUE TITLE
wrong date format displayed in blogposts

### ISSUE THREAD

**Issue by petsy** [finklabs/buccaneer#4]
Tested petsy_2014, bitbucket commit 1e17e39, Stand 8.2.2015

Example:
in blogpost: 2014-09-01 09:47
on website: MO 01 SEPTEMBER 2014

should be: September 1, 2014

**Comment by markfink** [comment]
this looks like an dateformat issue in your pelicanconfig.py. please check and let me know

**Comment by markfink** [comment]
like I said it is a config issue with your pelicanconfig.py: I added the following section:
DATE_FORMATS = {
    'en': '%B %d, %Y',
    'de': '%B %d, %Y',
}
please check and let me know if this is what you wanted.

**Comment by petsy** [comment]
this is what I would like and what I added to the configfile (different date formats for EN and DE, therefore I use locale):

DATE_FORMATS = {
    'en': ('en_US.utf8', '%B %d, %Y'),
    'de': ('de_DE.utf8', '%d. %B, %Y'),
}

I checked my list of available locale with command locale -a and the above are available. Still, it does not build the desired date formats.

The date format of the blogpost e.g. 2013-06-16 10:19 remains. 
It should be June 16, 2013 for EN and 16. Juni 2013 for DE.
